.github/workflows/setup.yml

```diff
@@ -0,0 +1,234 @@
+name: Setup tests
```

Comment:
Very important those tests. Really nice!

My only concern here is that this probably will be missed before the release, unless we decide to run the release always after Mondays. Is there any way to schedule this, also when any milestone is closed?

Reply:
We can schedule them another day  or run them manually.  Eventually we should be able to add them to the (semi?)automatic tests for the release.

Reply:
Thinking out of loud: We do have a target that is when a milestone is closed, that is something we have to do anyway. I would prefer this over running manually.

Reply:
We can use the milestone closing as a target, but we have to define when the milestone will be closed. Right now is up to the facilitator, sometime is before the release meeting, sometimes is after the release meeting and sometimes is during the meeting and the same is with the release itself and that can caused problems with tests. So IMO if we will be using this target we have to put it in the pre-release checks.

Reply:
@beraldoleal @richtja Thanks for the comments. IMHO this is something we should discuss at https://github.com/avocado-framework/avocado/issues/4692

Reply:
Hi @ana, my understand is that yes, this is also part of #4692, but there we are more concerned about the package creation/distribution itself.

So I'm not so sure about it. You are introducing a new test workflow here, and it is important to see the "best" schedule timing for it. Otherwise we are going to have the impression that all is covered, but in fact, any last minute change can break the installation because of timing. And since we usually fix things during the feature freeze at the end of week, this "timing" might be an issue.

While we could adopt what @richtja suggested adding a new item to our pre-release check, currently we are also trying to reduce the amount of manual checks before the release (#3456).

Reply:
@beraldoleal You're right, I liked the wrong issue 😅  I meant https://github.com/avocado-framework/avocado/issues/4601 that's indeed adding a new *automatic* pre-release.
I'm discussing this with @willianrampazzo on Monday, tell me if you want to join!